src/config-serializer/config-items/macro-action/EditableMacroAction.ts

```diff
@@ -0,0 +1,205 @@
+import {MacroAction, macroActionType} from './MacroAction';
+import {values as _values } from 'lodash';
+
+import {KeyAction, keyActionType} from '../key-action/KeyAction';
+import {KeystrokeAction} from '../key-action/KeystrokeAction';
+import {KeyMacroAction} from './KeyMacroAction';
+import {DelayMacroAction} from './DelayMacroAction';
+import {MouseButtonMacroAction} from './MouseButtonMacroAction';
+import {MoveMouseMacroAction} from './MoveMouseMacroAction';
+import {ScrollMouseMacroAction} from './ScrollMouseMacroAction';
+import {TextMacroAction} from './TextMacroAction';
+
+const macroActionTypeValues = _values(macroActionType);
```

Comment:
[This](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Object/values) should work instead.